PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,33 @@
-**Please check the type of change your PR introduces:**
-- [ ] Bugfix
-- [ ] Feature
-- [ ] Refactoring / Code style update (no functional changes, no api changes)
-- [ ] Build related changes
-- [ ] Documentation
-- [ ] Other (please describe): 
+### Describe the type of change your PR introduces:
 
+<!-- REMOVE ALL THAT DON'T APPLY -->
+- 🐛 Bugfix
+- ✨ Feature
+- 👾 Functional or Logic change
+- 🎨🖋 Appearance or Textual change (usually HTML/CSS changes)
+- 🏎 Performance improvement
+- 🚜 Refactoring (no change in the application is detected by the user)
+- 🚀 Build system
+- 📖 Documentation
+- ⚙️ Configuration
+- 🏗 Other (please describe)
 
-**Does this introduce a breaking change?**
+### Problem or Request:
+<!-- In one line of text, what was requested to be done. Attach a design screenshot if relevant. -->
+.
```

Comment:
Screenshots are below. I think that we should remove this from here

Reply:
This is the Request though, it could be performed in multiple ways. 
Examples:
- "Map is too slow"
- "Add chart X on page Y"